### DISCUSSION TITLE
How to reverse the order of blog pages, show oldest on top?

### DISCUSSION THREAD

**Discussion by mhmnbdllh** [datenstrom/community#363] [Ask a question] [answered]
First of all, I would like to thank to Yellow that it really makes everything easier. I would like to know how can I make all posts listed on Blog in reversed order. Is there any way to make the oldest post (on Blogpages) appears on top and the newest appears last? Thanks in advance.

**Comment by schulle4u** [comment, marked as answer]
Yes this is possible, though the blog plugin doesn't have a configuration option. Open `system/themes/snippets/content-blogpages.php` and find line 7: 

`<?php foreach($yellow->page->getPages() as $page): ?>`

Modify it as follows: 

`<?php foreach($yellow->page->getPages()->sort("published", true) as $page): ?>`

This will show you the blog pages in ascending order, sorted by publication date. 

HTH
Steffen

**Comment by mhmnbdllh** [comment]
Proud to be Yellow user. Thanks for your assistance @schulle4u.  It works!
